src/main/java/core/basesyntax/models/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.models;
+
+import core.basesyntax.models.painter.Color;
+import core.basesyntax.models.painter.ColorSupplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double PROPERTY_MULTIPLIER = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier(random);
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULT_COLOR.name(), 10);
+    }
+
+    public Figure getRandomFigure() {
+        int randFigureIndex = random.nextInt(Figures.values().length);
+        Figure figure = createFigure(Figures.values()[randFigureIndex]);
+        return figure;
```

Comment:
[Don’t create temporary variables if you can directly return value from the method](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method)

Reply:
My fault.
+fixed in new commit